docs/open_source/releases-and-branches.md

```diff
@@ -0,0 +1,73 @@
+Any other interim releases will be patch releases including any non-breaking / non-feature changes that can be
+cherry-picked from master in a straightforward manner.
+
+This is a change from our earlier process, where we simply tagged a release with whatever has been merged to master
+within the two-week cycle, and would rev the version number accordingly based on whether there were breaking changes.
+
+## Release Types
+
+### Breaking-change Releases
+
+These are currently our minor releases, e.g. 0.35.0, 0.36.0, etc. These follow the currently-outlined
+[Release Process](./release-process.md) and are performed against `master`.
+
+### Pre-releases
+
+MDC Web Catalog and MDC React both depend on MDC Web via npm dependencies. In order to be able to integrate
+breaking changes for the upcoming release ahead of time - preferably while the necessary changes are fresh in someone's
+mind - we will cut pre-releases of MDC Web when breaking changes are merged which warrant updates in the other
+repositories. These will be cut on an as-needed basis, with no set schedule.
+
+The Catalog and React repositories will each have a `next` branch, which will be updated to point to the pre-release.
+Pull requests should be filed against this branch for changes needed for the upcoming MDC Web release.
+
+Later, when the new version of MDC Web is released, the `next` branch will again be updated to point to the new release,
+and we will open a PR to squash & merge its contents onto `master`. Once this is done, the `next` branch will be reset
+against `master` to accommodate the subsequent release.
+
+The release process for pre-releases should be largely similar to for regular releases, with minor changes
+(such as the addition of a very important `lerna` option to avoid updating the `latest` tag on npm).
+
+Pre-releases will also involve updates to the CHANGELOG. We should consolidate these when tagging the final
+release to make it easier to read, since the pre-releases will be irrelevant at that point.
+
+### Bugfix Releases
+
+Bugfix releases for MDC Web will be released based on the last tagged release, with any docs/chores/fixes not
+involving breaking changes cherry-picked in. The process would look as follows (using 0.36.x as an example):
+
+1. `git checkout v0.36.0` (or e.g. `v0.36.1` if this is the second bugfix release since the last breaking-change release)
+2. `git cherry-pick -x <commits>`
+3. Follow the [release process](./release-process.md), choosing "patch" when lerna prompts for version
+4. Cherry-pick the publish and CHANGELOG commits onto `master` to ensure that bugfix releases are included in the overall history
+
+A branch can be created in/after step 1 if desired, but should be temporary - the bugfix release can be referenced via
+its git tag once the release process is done.
+
+We're planning to investigate scripting the process of finding non-breaking chores/fixes and cherry-picking them.
```

Comment:
I think refactor should also go in the list. As long as there are no breaking changes no functionality should be changing.